CWE/events/italy.txt

```diff
@@ -31,39 +31,47 @@ country_event = {
 
 #Italy becomes a republic
 country_event = {
-  id = 8007704
-  title = EVT_8007704_NAME
-  desc = EVT_8007704_DESC
-  picture = "nwo2_italian_constitutional_referendum"
-  is_triggered_only = yes
+	id = 8007704
+	title = EVT_8007704_NAME
+	desc = EVT_8007704_DESC
+	picture = "nwo2_italian_constitutional_referendum"
+	
+	is_triggered_only = yes
 
 	option = {
 		name = EVT_8007704_A
+		
 		government = democracy
+		
 		any_pop = { 
-				ideology = { factor = 0.20 value = conservative }
```

Comment:
why did you remove the boost for all ideologies? I put it there to make it so the pops have the same ideologies as the ones from the 1946 constituent assembly elections and so that communists have enough of a support base to represent OTL PCI popularity